_schemaV2.graphql

```diff
@@ -22198,15 +22153,21 @@ type Viewer {
     # Recaptcha token.
     recaptchaToken: String!
   ): VerifyUser
+
+  # Find a viewing room by ID
+  viewingRoom(id: ID!): ViewingRoom
   viewingRoomsConnection(
     after: String
+    before: String
+    featured: Boolean
     first: Int
+    ids: [ID!]
+    last: Int
     partnerID: ID
-    statuses: [ViewingRoomStatusEnum!]
+    statuses: [ViewingRoomStatusEnum!] = [live]
```

Comment:
Is `[live]` the new default?

Reply:
I was actually always a default: https://github.com/artsy/gravity/blob/main/app/graphql/types/query_type.rb#L97

but stitching code was not specifying it: https://github.com/artsy/metaphysics/blob/main/src/lib/stitching/gravity/v2/stitching.ts#L129

I understand it as this schema change shouldn't change anything, right? I will double-check

Reply:
yeah, I made such request:

```graphql
{
  viewer {
    viewingRoomsConnection(first: 1) {
      totalCount
    }
  }
}
```

And I see this is introspection:

```
{
            "query": "query ($_v0_first: Int, $_v0_statuses: [ViewingRoomStatusEnum!]) {\n  viewingRoomsConnection(first: $_v0_first, statuses: $_v0_statuses) {\n    totalCount\n  }\n}\n",
            "vars": {
              "_v0_first": 1,
              "_v0_statuses": [
                "**live**"
              ]
            }
          }
```

Reply:
Thanks for testing 🙏  Good the have the default in the schema.